src/Logging/StderrLoggerFactory.php

```diff
@@ -4,15 +4,19 @@
 namespace Momento\Logging;
 
 use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
 
 /**
- * Factory class for returning a StderrEchoLogger instance.
+ * Factory class for returning a StderrLogger instance.
  */
-class StderrLoggerFactory implements ILoggerFactory
+class StderrLoggerFactory extends LoggerFactoryBase
 {
+    public function __construct(?string $logLevel=LogLevel::INFO) {
```

Comment:
it's fine to require log level here if it's appropriate for this logger implementation.  doesn't mean it needs to be in the factory interface.

take all this with a grain of salt, you're more familiar with the PHP ecosystem than I am, but the advice i'm giving here was definitely the way to go for the other languages I've touched so far.

Reply:
Yep, it's here and defaulted to INFO as an implementation detail of this class only.